index.css

```diff
@@ -0,0 +1,140 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+
+* {
+  font-family: sans-serif;
+}
+
+body {
+  margin: 5px;
+  padding: 5px;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+body::before {
+  content: "";
```

Comment:
- [ ] I am unable to click the form inputs for adding books. There is a CSS issue. Kindly see the video demo below and how to fix this.

  [screen-capture (1).webm](https://user-images.githubusercontent.com/36542986/227984578-e4306021-aab5-4b5f-bbf4-edf5ce4690c1.webm)